vwb/app/views/users/registration.html.erb

```diff
@@ -0,0 +1,54 @@
+<h1>Registration</h1>
+
+<%= form_with model: @user do |form| %>
+    <div>
+        <%= form.label :email %><br>
+        <%= form.email_field :email, value: current_userlogin.email, :readonly => true%>
+    </div>
+
+    <div>
+        <%= form.label :firstName %><br>
+        <%= form.text_field :firstName %>
+        <% @user.errors.full_messages_for(:firstName).each do |message| %>
+            <div><%= message %></div>
+        <% end %>
+    </div>
+
+    <div>
+        <%= form.label :lastName %><br>
+        <%= form.text_field :lastName %>
+        <% @user.errors.full_messages_for(:lastName).each do |message| %>
+            <div><%= message %></div>
+        <% end %>
+    </div>
+    
+    <div>
+        <%= form.label :role %><br>
+        <%= form.select :role, options_for_select([["Member", 0], ["Admin", 1]]), :prompt => 'Select Role' %>
+    </div>

```

Comment:
Should probably remove selecting a role in the registration form, and just default to member